packages/at_secondary_proxy/README.md

```diff
@@ -37,6 +48,8 @@ All of our software is open with intent. We welcome contributions - we want pull
 to hear about issues. See also [CONTRIBUTING.md](../CONTRIBUTING.md)
 
 ## What's here / changelog
+### May 30th 2025
+* Updated README
```

Comment:
If we just say "Updated README" every time the README is updated to reflect new stuff then this section doesn't really provide the reader what it could/should.

This is a pretty major change, and deserves some attention in the changelog.

Reply:
OK sounds good

Reply:
@cconstab thanks for updating the changelog. I'd suggest just deleting this section of the README, and maybe linking to the changelog.